edx_rest_framework_extensions/auth/jwt/authentication.py

```diff
@@ -187,4 +187,4 @@ def get_decoded_jwt_from_auth(request):
     if not is_jwt_authenticated(request):
         return None
 
-    return jwt_decode_handler(request.auth)
+    return configured_jwt_decode_handler(request.auth)
```

Comment:
**Reviewer Note:** This wasn't causing a known problem, but I updated it for consistency.  I did not add a special test for this.